plugins/modules/identity/keycloak/keycloak_authentication.py

```diff
@@ -0,0 +1,289 @@
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: keycloak_authentication
+short_description: Configure authentication in Keycloak
+description:
+    - This module actually can only make a copy of an existing authentication flow, add an execution to it and configure it.
+    - It can also delete the flow.
+version_added: "3.0.0"
+options:
+    realm:
+        description:
+            - The name of the realm in which is the authentication.
+        required: true
+        type: str
+    alias:
+        description:
+            - Alias for the authentication flow
+        required: true
+        type: str
+    description:
+        description:
+            - Description of the flow.
+        required: false
+        type: str
+    providerId:
+        description:
+            - providerId for the new flow when not copied from an existing flow.
+        required: false
+        type: str
+    copyFrom:
+        description:
+            - flowAlias of the authentication flow to use for the copy.
+        required: false
+        type: str
+    authenticationExecutions:
+        description:
+            - Configuration structure for the executions.
+        required: false
+        type: list
+        elements: str
+        suboptions:
+            providerId:
+                description:
+                    - providerId for the new flow when not copied from an existing flow.
+                required: false
+                type: str
+            displayName:
+                description:
+                    - Name of the execution or subflow to create or update-
+                required: false
+                type: str
+            requirement:
+                description:
+                    - Control status of the subflow or execution.
+                choices: [ "REQUIRED", "ALTERNATIVE", "DISABLED", "CONDITIONAL" ]
+                required: false
+                type: str
+            flowAlias:
+                description:
+                    - Alias of parent flow
+                required: false
+                type: str
+    state:
+        description:
+            - Control if the authentication flow must exists or not
+        choices: [ "present", "absent" ]
+        default: present
+        required: false
+        type: str
+    force:
+        type: bool
+        default: false
+        description:
+            - If true, allows to remove the authentication flow and recreate it.
+        required: false
+extends_documentation_fragment:
+- community.general.keycloak
+
+author:
+    - Philippe Gauthier (@elfelip)
+    - Gaëtan Daubresse (@Gaetan2907)
+'''
+
+EXAMPLES = '''
+    - name: Create an authentication flow from first broker login and add an execution to it.
+      keycloak_authentication:
```

Comment:
```suggestion
      community.general.keycloak_authentication:
```
Always use FQCN in docs.